app/models/registration/requests/RequestDetailWithoutId.scala

```diff
@@ -34,3 +34,9 @@ object RequestDetailWithoutId {
       }
   }
 }
+
+case class Contact(email: String, phone: Option[String])
```

Comment:
Rename:
* Contact -> ContactDetails
* email -> emailAddress
* phone -> phoneNumber
This way you can remove the transformation from my previous message.

Reply:
As above.